Source/Core/DiscIO/VolumeCreator.cpp

```diff
@@ -217,8 +215,6 @@ EDiscType GetDiscType(IBlobReader& _rReader)
 	u32 GCMagic = Reader.Read32(0x1C);
 
 	// check for Wii
-	if (WiiMagic == 0x5D1C9EA3 && WiiContainerMagic != 0)
-		return DISC_TYPE_WII;
 	if (WiiMagic == 0x5D1C9EA3 && WiiContainerMagic == 0)
```

Comment:
There are various other changes I'd want to make to clean this up before it was merged. This is mostly just a test to get some feedback if anyone knew what this check was actually for, and if it would break anything